include/rocksdb/wal_filter.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2013, Facebook, Inc.  All rights reserved.
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
+// Copyright (c) 2013 The LevelDB Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+#ifndef STORAGE_ROCKSDB_INCLUDE_WAL_FILTER_H_
+#define STORAGE_ROCKSDB_INCLUDE_WAL_FILTER_H_
+
+#ifndef ROCKSDB_LITE
```

Comment:
Guard should be avoided in public header. Maybe just leave it in the header files.